src/stack.js

```diff
@@ -15,21 +15,22 @@ export default class Stack {
 
   // this.pop = find length of array and make sure length = -1 and it was taken off of the beginning
   pop() {
-    this.elements.shift()
+    const value = this.elements.shift()
```

Comment:
You need to decide where the top of the stack is.  You are pushing to the end of the array but popping and peeking from the beginning of the array.  The result is going to be "First In First Out" which is a queue, not a stack.  (A stack is "Last In First Out".  Think of a stack of plates - the last one put on is the first one taken off.)